### PR TITLE
Update branch

### PR DESCRIPTION
**Pull request by SirriRyisa**
Specifically created for pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,19 +4,19 @@ This is a React module capstone project. It is built with react and redux by usi
 This application shows the statistics of covid cases in various countries.
 ## Screenshot
 
-![screenshot]()
+![screenshot](https://user-images.githubusercontent.com/38283436/159509800-f54c6d0c-5a1a-4126-8148-c0420d317c64.png)
 
 ## Live Demo
 
-[Live Link]()
+[Live Link](https://sirriryisa.github.io/covid-19-data/)
```
Comment (uwadonat) [inline README.md]: - [ ] This link should be pointing to the application deployed using Heroku, Netlify, or GitHub Pages. Now it is pointing to your Readme file.

File: README.md [hunk 2]
```diff
@@ -4,19 +4,19 @@ This is a React module capstone project. It is built with react and redux by usi
...
 
-[Video Presentation Link]()
+[Video Presentation Link](https://www.loom.com/share/ff960555d6474dff94ae12c37fc34e7c)
```
Comment (uwadonat) [inline README.md:13]: - [ ] Kindly add in your presentation a highlight about a piece of the code you find interesting or you are proud of making.

File: README.md [hunk 3]
```diff
@@ -4,19 +4,19 @@ This is a React module capstone project. It is built with react and redux by usi
...
 
 ## Built With
 
 - React
 - Redux
-- Jest
+- Jes
 
 ## Getting Started
```
Comment (uwadonat) [inline README.md]: - [ ] Let's add instructions to run the test in your project.

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **[SirriRyisa](https://github.com/SirriRyisa),** :hand: 

Good job so far! :mechanical_arm: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR message is very descriptive ✔️
- You implemented well JavaScript best practices ✔️
-  No linter errors ✔️

almost

## Status:Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @SirriRyisa,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**